HttpFs/HttpFs.fs

```diff
@@ -1060,16 +1060,25 @@ module Client =
         body             = response.GetResponseStream()
         luggage          = Some (upcast response) }
 
-  /// Sends the HTTP request and returns the full response as a Response record, asynchronously.
-  let getResponse request = job {
-    let webRequest, exec = toHttpWebRequest HttpFsState.empty request
-    do! exec
-    let! resp = getResponseNoException webRequest
-    let wrapped =
+  let tryGetResponse request =
```

Comment:
The commit point is when the Alt produced by `prepareJob` is committed to, Thinks done in `prepareJob` are done before the `Alt` becomes ready; they instantiate the `Alt`. The commit point is when `tryEndRequest`, called by `Alt.fromBeginEnd`, returns.